<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>536</width>
    <height>599</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>362</width>
      <height>473</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTableWidget" name="sudoku_board">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>360</width>
         <height>270</height>
        </size>
       </property>
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <attribute name="horizontalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderMinimumSectionSize">
        <number>40</number>
       </attribute>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>40</number>
       </attribute>
       <attribute name="horizontalHeaderHighlightSections">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <row>
        <property name="text">
         <string/>
        </property>
       </row>
       <row>
        <property name="text">
         <string/>
        </property>
       </row>
       <row>
        <property name="text">
         <string/>
        </property>
       </row>
       <row>
        <property name="text">
         <string/>
        </property>
       </row>
       <row>
        <property name="text">
         <string/>
        </property>
       </row>
       <row>
        <property name="text">
         <string/>
        </property>
       </row>
       <row>
        <property name="text">
         <string/>
        </property>
       </row>
       <row>
        <property name="text">
         <string/>
        </property>
       </row>
       <row>
        <property name="text">
         <string/>
        </property>
       </row>
       <column>
        <property name="text">
         <string/>
        </property>
       </column>
       <column>
        <property name="text">
         <string/>
        </property>
       </column>
       <column>
        <property name="text">
         <string/>
        </property>
       </column>
       <column>
        <property name="text">
         <string/>
        </property>
       </column>
       <column>
        <property name="text">
         <string/>
        </property>
       </column>
       <column>
        <property name="text">
         <string/>
        </property>
       </column>
       <column>
        <property name="text">
         <string/>
        </property>
       </column>
       <column>
        <property name="text">
         <string/>
        </property>
       </column>
       <column>
        <property name="text">
         <string/>
        </property>
       </column>
       <item row="0" column="0">
        <property name="text">
         <string>0</string>
        </property>
       </item>
       <item row="0" column="1">
        <property name="text">
         <string>0</string>
        </property>
       </item>
       <item row="0" column="2">
        <property name="text">
         <string>0</string>
        </property>
       </item>
       <item row="0" column="3">
        <property name="text">
         <string>0</string>
        </property>
       </item>
       <item row="0" column="4">
        <property name="text">
         <string>0</string>
        </property>
       </item>
       <item row="0" column="5">
        <property name="text">
         <string>0</string>
        </property>
       </item>
       <item row="0" column="6">
        <property name="text">
         <string>0</string>
        </property>
       </item>
       <item row="0" column="7">
        <property name="text">
         <string>0</string>
        </property>
       </item>
       <item row="0" column="8">
        <property name="text">
         <string>0</string>
        </property>
       </item>
       <item row="1" column="0">
        <property name="text">
         <string>0</string>
        </property>
       </item>
       <item row="1" column="1">
        <property name="text">
         <string>0</string>
        </property>
       </item>
       <item row="1" column="2">
        <property name="text">
         <string>0</string>
        </property>
       </item>
       <item row="1" column="3">
        <property name="text">
         <string>0</string>
        </property>
       </item>
       <item row="1" column="4">
        <property name="text">
         <string>0</string>
        </property>
       </item>
       <item row="1" column="5">
        <property name="text">
         <string>0</string>
        </property>
       </item>
       <item row="1" column="6">
        <property name="text">
         <string>0</string>
        </property>
       </item>
       <item row="1" column="7">
        <property name="text">
         <string>0</string>
        </property>
       </item>
       <item row="1" column="8">
        <property name="text">
         <string>0</string>
        </property>
       </item>
       <item row="2" column="0">
        <property name="text">
         <string>0</string>
        </property>
       </item>
       <item row="2" column="1">
        <property name="text">
         <string>0</string>
        </property>
       </item>
       <item row="2" column="2">
        <property name="text">
         <string>0</string>
        </property>
       </item>
       <item row="2" column="3">
        <property name="text">
         <string>0</string>
        </property>
       </item>
       <item row="2" column="4">
        <property name="text">
         <string>0</string>
        </property>
       </item>
       <item row="2" column="5">
        <property name="text">
         <string>0</string>
        </property>
       </item>
       <item row="2" column="6">
        <property name="text">
         <string>0</string>
        </property>
       </item>
       <item row="2" column="7">
        <property name="text">
         <string>0</string>
        </property>
       </item>
       <item row="2" column="8">
        <property name="text">
         <string>0</string>
        </property>
       </item>
       <item row="3" column="0">
        <property name="text">
         <string>0</string>
        </property>
       </item>
       <item row="3" column="1">
        <property name="text">
         <string>0</string>
        </property>
       </item>
       <item row="3" column="2">
        <property name="text">
         <string>0</string>
        </property>
       </item>
       <item row="3" column="3">
        <property name="text">
         <string>0</string>
        </property>
       </item>
       <item row="3" column="4">
        <property name="text">
         <string>0</string>
        </property>
       </item>
       <item row="3" column="5">
        <property name="text">
         <string>0</string>
        </property>
       </item>
       <item row="3" column="6">
        <property name="text">
         <string>0</string>
        </property>
       </item>
       <item row="3" column="7">
        <property name="text">
         <string>0</string>
        </property>
       </item>
       <item row="3" column="8">
        <property name="text">
         <string>0</string>
        </property>
       </item>
       <item row="4" column="0">
        <property name="text">
         <string>0</string>
        </property>
       </item>
       <item row="4" column="1">
        <property name="text">
         <string>0</string>
        </property>
       </item>
       <item row="4" column="2">
        <property name="text">
         <string>0</string>
        </property>
       </item>
       <item row="4" column="3">
        <property name="text">
         <string>0</string>
        </property>
       </item>
       <item row="4" column="4">
        <property name="text">
         <string>0</string>
        </property>
       </item>
       <item row="4" column="5">
        <property name="text">
         <string>0</string>
        </property>
       </item>
       <item row="4" column="6">
        <property name="text">
         <string>0</string>
        </property>
       </item>
       <item row="4" column="7">
        <property name="text">
         <string>0</string>
        </property>
       </item>
       <item row="4" column="8">
        <property name="text">
         <string>0</string>
        </property>
       </item>
       <item row="5" column="0">
        <property name="text">
         <string>0</string>
        </property>
       </item>
       <item row="5" column="1">
        <property name="text">
         <string>0</string>
        </property>
       </item>
       <item row="5" column="2">
        <property name="text">
         <string>0</string>
        </property>
       </item>
       <item row="5" column="3">
        <property name="text">
         <string>0</string>
        </property>
       </item>
       <item row="5" column="4">
        <property name="text">
         <string>0</string>
        </property>
       </item>
       <item row="5" column="5">
        <property name="text">
         <string>0</string>
        </property>
       </item>
       <item row="5" column="6">
        <property name="text">
         <string>0</string>
        </property>
       </item>
       <item row="5" column="7">
        <property name="text">
         <string>0</string>
        </property>
       </item>
       <item row="5" column="8">
        <property name="text">
         <string>0</string>
        </property>
       </item>
       <item row="6" column="0">
        <property name="text">
         <string>0</string>
        </property>
       </item>
       <item row="6" column="1">
        <property name="text">
         <string>0</string>
        </property>
       </item>
       <item row="6" column="2">
        <property name="text">
         <string>0</string>
        </property>
       </item>
       <item row="6" column="3">
        <property name="text">
         <string>0</string>
        </property>
       </item>
       <item row="6" column="4">
        <property name="text">
         <string>0</string>
        </property>
       </item>
       <item row="6" column="5">
        <property name="text">
         <string>0</string>
        </property>
       </item>
       <item row="6" column="6">
        <property name="text">
         <string>0</string>
        </property>
       </item>
       <item row="6" column="7">
        <property name="text">
         <string>0</string>
        </property>
       </item>
       <item row="6" column="8">
        <property name="text">
         <string>0</string>
        </property>
       </item>
       <item row="7" column="0">
        <property name="text">
         <string>0</string>
        </property>
       </item>
       <item row="7" column="1">
        <property name="text">
         <string>0</string>
        </property>
       </item>
       <item row="7" column="2">
        <property name="text">
         <string>0</string>
        </property>
       </item>
       <item row="7" column="3">
        <property name="text">
         <string>0</string>
        </property>
       </item>
       <item row="7" column="4">
        <property name="text">
         <string>0</string>
        </property>
       </item>
       <item row="7" column="5">
        <property name="text">
         <string>0</string>
        </property>
       </item>
       <item row="7" column="6">
        <property name="text">
         <string>0</string>
        </property>
       </item>
       <item row="7" column="7">
        <property name="text">
         <string>0</string>
        </property>
       </item>
       <item row="7" column="8">
        <property name="text">
         <string>0</string>
        </property>
       </item>
       <item row="8" column="0">
        <property name="text">
         <string>0</string>
        </property>
       </item>
       <item row="8" column="1">
        <property name="text">
         <string>0</string>
        </property>
       </item>
       <item row="8" column="2">
        <property name="text">
         <string>0</string>
        </property>
       </item>
       <item row="8" column="3">
        <property name="text">
         <string>0</string>
        </property>
       </item>
       <item row="8" column="4">
        <property name="text">
         <string>0</string>
        </property>
       </item>
       <item row="8" column="5">
        <property name="text">
         <string>0</string>
        </property>
       </item>
       <item row="8" column="6">
        <property name="text">
         <string>0</string>
        </property>
       </item>
       <item row="8" column="7">
        <property name="text">
         <string>0</string>
        </property>
       </item>
       <item row="8" column="8">
        <property name="text">
         <string>0</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QGridLayout" name="gridLayout">
           <item row="1" column="1">
            <widget class="QPushButton" name="keypad_button_5">
             <property name="text">
              <string>5</string>
             </property>
             <attribute name="buttonGroup">
              <string notr="true">keypad_group</string>
             </attribute>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QPushButton" name="keypad_button_3">
             <property name="text">
              <string>3</string>
             </property>
             <attribute name="buttonGroup">
              <string notr="true">keypad_group</string>
             </attribute>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QPushButton" name="keypad_button_1">
             <property name="text">
              <string>1</string>
             </property>
             <attribute name="buttonGroup">
              <string notr="true">keypad_group</string>
             </attribute>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QPushButton" name="keypad_button_8">
             <property name="text">
              <string>8</string>
             </property>
             <attribute name="buttonGroup">
              <string notr="true">keypad_group</string>
             </attribute>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QPushButton" name="keypad_button_2">
             <property name="text">
              <string>2</string>
             </property>
             <attribute name="buttonGroup">
              <string notr="true">keypad_group</string>
             </attribute>
            </widget>
           </item>
           <item row="2" column="2">
            <widget class="QPushButton" name="keypad_button_9">
             <property name="text">
              <string>9</string>
             </property>
             <attribute name="buttonGroup">
              <string notr="true">keypad_group</string>
             </attribute>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QPushButton" name="keypad_button_6">
             <property name="text">
              <string>6</string>
             </property>
             <attribute name="buttonGroup">
              <string notr="true">keypad_group</string>
             </attribute>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QPushButton" name="keypad_button_7">
             <property name="text">
              <string>7</string>
             </property>
             <attribute name="buttonGroup">
              <string notr="true">keypad_group</string>
             </attribute>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QPushButton" name="keypad_button_4">
             <property name="text">
              <string>4</string>
             </property>
             <attribute name="buttonGroup">
              <string notr="true">keypad_group</string>
             </attribute>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="erase_button">
           <property name="text">
            <string>Erase</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">keypad_group</string>
           </attribute>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QPushButton" name="solve_button">
           <property name="text">
            <string>Solve</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton">
           <property name="text">
            <string>New Game</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>536</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="keypad_group"/>
 </buttongroups>
</ui>
